scripts/lease.sh

```diff
@@ -2,24 +2,35 @@
 
 lease() {
   local tries=1
+  local cluster_args
+
+  # place priority over cluster regex (name)
+  if [ -n "${CLUSTER_REGEX}" ]; then
+    cluster_args="--cluster-regex=${CLUSTER_REGEX} "
```

Comment:
Dont append a space at the end here. It makes the cluster args variable below harder to read because you rely on that space.